src/ape_plugins/_cli.py

```diff
@@ -22,38 +22,89 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _remove_prefix_add_version(set_convert, _prefix, version):
```

Comment:
```suggestion
def _format_output(output_set, _prefix, version):
```

Two things:

1. This method indicates that it violates the single-responsibility principle.. You can tell by its name `_remove_prefix_add_version`, that is like two method names combined into one.

2. `set_convert` sounds like a method name